### ISSUE TITLE
A11y_VisualStudioCodeClient_Terminal_ScreenReader: In accessible terminal, NVDA does not announce the command when the focus reaches onto any line of the command's output.

### ISSUE THREAD

**Issue by ManishaC1** [microsoft/vscode#187675]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver".**

## GitHub bugs
#A11ySev4; #A11yUsable; #A11yTCS; #DesktopApp; #Visual Studio Code Client; #DesktopApp; #BM-VisualStudioCodeClient-Win32-Aug2022; #NVDA; #Win32; #Win11; #A11yeDAD; #ScreenReader; #FTP; #STP; #JAWS;

## Actual:

In Accessible terminal, NVDA does not announce the command when the focus reaches onto any line of the command's output.

## Expected:

NVDA should announce the executed command when the focus moves to any line of the command's output, allowing users to understand and track the executed commands.

## Environment Details:​

Application Name: Visual Studio Code
Version: 1.81.0-insider (user setup)
V8: 10.8.168.25-electron.0
Windows 11 22H2 OS build 22621.1848
Screen Reader: NVDA (2023.1)/JAWS

## Repro Steps:

1. Launch Visual Studio Code.
2. Tab to Terminal.
3. Execute any command, that produces correct output.
4. Turn on NVDA/JAWS
5. Then use Shift+Tab shortcut key to switch to "Accessible Terminal".
6. Move NVDA focus onto any line of the commands output.
7. Check if NVDA announcing anything about the command when the focus is on its output.

## User Impact:

This issue affects the user experience of the terminal for NVDA users, especially those relying on auditory feedback.

"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

## Attachments:

https://github.com/microsoft/vscode/assets/105198219/fbfceecb-5516-4d69-ae5c-110d5c612d46

https://github.com/microsoft/vscode/assets/105198219/85bdc5c8-f6d9-4261-a3ef-47508afee07e

**Comment by meganrogge** [comment]
This is working for me on macOS with VoiceOver. I'll try windows

**Comment by meganrogge** [comment]
@ManishaC1 is your screen reader in browse or focus mode?

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by ManishaC1** [comment]
Please reopen the issue as the issue is not resolved

**Comment by meganrogge** [comment]
@ManishaC1 pls provide a gif

**Comment by ManishaC1** [comment]
@meganrogge 

Environment Details:

Version: 1.83.0-insider (user setup)
Commit: b257cce9e7fa2d5f475ac16c15b40280d0b97957
Date: 2023-09-08T17:37:16.115Z
Electron: 25.8.0
ElectronBuildId: 23503258
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22621


The issue was, NVDA in browse mode when focus move onto any line of the executed command output in "Accessible Terminal", fails to announce the command. But now in the above environment, we are not able to switch to "Accessible Terminal" using "Shift+ tab" shortcut key. Can you please tell me how to switch to "Accessible terminal".

https://github.com/microsoft/vscode/assets/105198219/02302030-2b9d-4406-a260-68afa178540c

#PendingInput; #Regressed:09-11-23

**Comment by meganrogge** [comment]
@ManishaC1 you should know this via the aria label which provies a hint about how to access it, but now that's accessed via `alt+f2` or `ctrl/cmd+up arrow`

**Comment by ManishaC1** [comment]
@meganrogge 
Environment Detials:
Version: 1.83.0-insider (user setup)
Commit: b257cce9e7fa2d5f475ac16c15b40280d0b97957
Date: 2023-09-08T17:37:16.115Z
Electron: 25.8.0
ElectronBuildId: 23503258
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.22621

Below is the video of the issue. Please reactivate

https://github.com/microsoft/vscode/assets/105198219/bc3d0543-62f6-4cb2-8019-b39c84df5ab7

**Comment by meganrogge** [comment]
You're in the terminal accessible view there. It's not expected to work there. Use the normal terminal buffer and press up arrow
